### PR TITLE
Upgrade antrea version to v1.12

### PR DESCRIPTION
**Pull request by reachjainrahul**

### CODE REVIEW

File: pkg/apiserver/registry/inventory/virtualmachine/rest.go [hunk 1]
```diff
@@ -64,6 +64,9 @@ func (r *REST) New() runtime.Object {
 	return &runtimev1alpha1.VirtualMachine{}
 }
 
+func (r *REST) Destroy() {
```
Comment (shenmo3) [inline pkg/apiserver/registry/inventory/virtualmachine/rest.go:67]: Is this a new requirement for upgrading?

**Review by shenmo3 — APPROVED** [review]
/LGTM with a minor question.

File: pkg/apiserver/registry/inventory/virtualmachine/rest.go (hunk 1 above)
Comment (reachjainrahul) [reply pkg/apiserver/registry/inventory/virtualmachine/rest.go:67]: Yes.. with antrea 1.12, runtime version is upgraded too.

**Review by shenmo3 — APPROVED** [review]

**Pull request merged**